BrainPortal/public/swagger/cbrain-4.5.1-swagger.yaml

```diff
@@ -0,0 +1,258 @@
+# Swagger description of the CBRAIN API
+# Pierre Rioux, August 2016 - October 2016
+swagger: '2.0'
+info:
+  title: CBRAIN API
+  description: Interface to control CBRAIN operations
+  version: "4.5.1"
+# The domain of the service
+host: portal.cbrain.mcgill.ca
+schemes:
+  - https
+basePath: /
+produces:
+  - application/json
+paths:
+#=============================================================================
+# SESSIONS ACTIONS
+#=============================================================================
+  /session/new:
+    get:
+      summary: New session initiator
+      description: |
+        This returns an object with the information necessary to
+        create a new session.
+      tags:
+        - Sessions
+      responses:
+        200:
+          description: An object with an authenticity token
+          schema:
+            type: object
+            properties:
+              authenticity_token:
+                type: string
+#=============================================================================
+  /session:
+    post:
+      summary: Create a new session
+      description: |
+        This is the main entry point to create a CBRAIN session.
+      parameters:
+        - name: authenticity_token
+          in: query
+          description: The token returned by /sessions/new
+          required: true
+          type: string
+        - name: login
+          in: query
+          description: The CBRAIN user's username
+          required: true
+          type: string
+        - name: password
+          in: query
+          description: The CBRAIN user's password
+          required: true
+          type: string
```

Comment:
In Swagger is there no password type? Could this have a security impact if users are trying out our API with their passwords?